contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -37,6 +37,129 @@
     "email",
 ]
 
+CANONICAL_COLUMN_NAMES_CORE = [
```

Comment:
Why not use sets directly here? (Also applies to other global variables below)

Reply:
This is a good idea. Will do! Thanks for the review.

I used list before I created test cases using sets. I did not foresee what I was going to do far enough :)